docs/en/patterns/aside.md

```diff
@@ -5,15 +5,14 @@ table_of_contents: false
 
 ## Aside
 
+<hr>
```

Comment:
We don't usually have an HR after the heading

Reply:
I added to provide a divide from heading and content 😄